auth/authenticate.js

```diff
@@ -1,20 +1,24 @@
+require('dotenv').config();
 const jwt = require('jsonwebtoken');
 
 const jwtKey =
   process.env.JWT_SECRET ||
   'add a .env file to root of project with the JWT_SECRET variable';
 
-// quickly see what this file exports
+
 module.exports = {
-  authenticate,
+  authenticate, 
 };
 
-// implementation details
+
 function authenticate(req, res, next) {
   const token = req.get('Authorization');
+  const secret = jwtKey 
```

Comment:
this one isn't needed. You could have done just jwtKey, but I know you were dealing some bugs